tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -219,6 +234,12 @@ sub install_kubevirt_packages {
     zypper_call("--gpg-auto-import-keys ref");
 
     my $virt_manifests = 'containerized-data-importer-manifests kubevirt-manifests kubevirt-virtctl';
+    my $virt_tests = 'kubevirt-tests';
+    my $container_tag = get_var("CONTAINER_TAG");
+    if ($container_tag) {
```

Comment:
If necessary, seems no need to check this if?

Reply:
Yes, you're right.